app/templates/gentelella/admin/event/tickets/discount_codes_create.html

```diff
@@ -254,6 +259,10 @@
             $newDiscountForm.validator('validate');
         });
 
+        var host = (document.URL.split("//")[1].split("/"))[0];
+        $("#code").change(function(){
+            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?code="+$(this).val());
```

Comment:
@magdalenesuo 

F.Y.I a much cleaner way would be

```javascript
$("#link").val(window.location.protocol + "//" + window.location.host + "/e/" + "{{ event.identifier }}" + "/?code=" + $(this).val());
```

You need not change this now ... But just keep this in mind :wink: